v2/protocol/http/write_request.go

```diff
@@ -86,7 +86,7 @@ func (b *httpRequestWriter) setBody(body io.Reader) error {
 		default:
 			// This is where we'd set it to -1 (at least
 			// if body != NoBody) to mean unknown, but
-			// that broke people during the Go 1.8 testing
+			// that broke people during the Go 1.8 testin2
```

Comment:
Typo?

Reply:
yeah, but not by me :)

I need to file a README update, lemme quickly address

Reply:
@duglin fixed